src/main/java/world/bentobox/bentobox/database/json/JSONDatabaseHandler.java

```diff
@@ -112,6 +112,7 @@ public T loadObject(@NonNull String uniqueId) {
             return completableFuture;
         }
         String path = DATABASE_FOLDER_NAME + File.separator + dataObject.getSimpleName();
+        String backupPath = DATABASE_FOLDER_NAME + "_backup" + File.separator + dataObject.getSimpleName();
```

Comment:
There is now a new JSON backup database folder for files that have been saved over. Although this will provide a backup of files, if the admin repeatedly runs the server when files are damaged, even these backups will be overwritten. Therefore, I am not sure how useful this will be. If the admin is smart and sees the issue, they will read protect this folder and use it to copy back files.